doc/examples/color_exposure/plot_colormap.py

```diff
@@ -0,0 +1,30 @@
+"""
+================================
+Drawing an image with a colormap
+================================
+
+This example demonstates the use of the :py:func:`skimage.color.img_to_cmap`
+function, which takes an input grayscale image and returns an RGB image drawn
+by using the specified colormap.
+"""
+from skimage import color, data
+from matplotlib import pyplot as plt
+from matplotlib import colors
+
+
+img = data.camera()
+out = color.img_to_cmap(img, 'viridis')
```

Comment:
@stefanv 
I just realized that this is a horrible mistake, I should be using the computed output and not pass the gray image to `imshow`